test/e2e/kappcontroller/config_test.go

```diff
@@ -337,3 +337,88 @@ stringData:
 		assert.Contains(t, cr.Status.Fetch.Stderr, "MANIFEST_UNKNOWN: manifest unknown;")
 	})
 }
+
+func TestConfig_PackageInstallDefaultSyncPeriod(t *testing.T) {
+	env := e2e.BuildEnv(t)
+	logger := e2e.Logger{}
+	kapp := e2e.Kapp{T: t, Namespace: env.Namespace, L: logger}
+	sas := e2e.ServiceAccounts{Namespace: env.Namespace}
+	kubectl := e2e.Kubectl{T: t, Namespace: env.Namespace, L: logger}
+
+	configName := "test-config-package-install-default-sync-period-config"
+	name := "install-pkg-test"
+
+	packageInstallYaml := fmt.Sprintf(`
+---
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageRepository
```

Comment:
_non-blocker_: I think we can remove the `PackageRepository` in this and the other e2e test.

Reply:
Removed `PackageRepository` from the newly added e2e tests.

Reply:
I tried removing `PackageRepository` but it fails probably because `PacakgeRepository` provisioned by other tests gets cleaned up. I think we would need them.

https://github.com/carvel-dev/kapp-controller/actions/runs/5453108138/jobs/9921523639#step:6:4315